terraform/modules/iam_role_policy/aws_broker/policy.json

```diff
@@ -26,6 +25,15 @@
         "arn:${aws_partition}:rds:${aws_default_region}:${account_id}:subgrp:${rds_subgroup}"
       ]
     },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "rds:DescribeDBParameterGroups"
+      ],
+      "Resource": [
+        "arn:${aws_partition}:rds:${aws_default_region}:${account_id}:pg:*"
```

Comment:
Just double checking, which resource is this that's being granted permissions for this action?  Also, removing the `rds:DescribeDBParameterGroups` from the other resource won't cause any other issues?

Reply:
This is granting `rds:DescribeDBParameterGroups` on all parametergroups in our account

Reply:
Thanks!